### PR TITLE
NCDA Report - Nutrition Behavior pane

### PR DESCRIPTION
**Pull request by anvmn**
#519 
#534

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_1229

**Comment by anvmn** [comment]
Selection_1231

**Comment by anvmn** [comment]
Peek 2022-11-06 20-34

**Comment by anvmn** [comment]
@AronNovak please review

File: server/RoboFile.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ public function deployPantheon($branchName = 'master') {
 
     if ($result->getMessage()) {
       $this->yell($result->getMessage());
-      throw new Exception('The GitHub working directory is dirty. Please commit any pending changes or add to .gitignore.');
+      // throw new Exception('The GitHub working directory is dirty. Please commit any pending changes or add to .gitignore.');
```
Comment (AronNovak) [inline server/RoboFile.php]: We should solve it in a different way, we should keep the working directory clean simply.

**Review by AronNovak — APPROVED** [review]
Looks good, except the deploy code part.

File: server/RoboFile.php (hunk 1 above)
Comment (anvmn) [reply server/RoboFile.php]: This was never meant to be commited. Reverted.

**Pull request merged**